webdriver/tests/bidi/script/evaluate/await_promise.py

```diff
@@ -18,6 +21,30 @@ async def test_await_delayed_promise(bidi_session, top_context):
     assert result == {"type": "string", "value": "SOME_DELAYED_RESULT"}
 
 
+@pytest.mark.asyncio
+async def test_await_rejected_promise(bidi_session, top_context):
```

Comment:
Mind renaming to `test_await_promise_rejected` so that we can move the test `test_await_resolved_promise` right next to it as `test_await_promise_resolved`?

Reply:
All the promise tests have been renamed `test_(no_)await_promise_(delayed|rejected|resolved)`